code/modules/admin/verbs/randomverbs.dm

```diff
@@ -549,6 +549,32 @@ Traitors and the like can also be revived with the previous role mostly intact.
 	admin_ticket_log(M, msg)
 	SSblackbox.record_feedback("tally", "admin_verb", 1, "Rejuvinate") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+/client/proc/cmd_admin_freeze(mob/living/M in GLOB.mob_list)
+	set category = "Special Verbs"
```

Comment:
We don't have this category, put it under Admin or Adminbus

Reply:
> We don't have this category, put it under Admin or Adminbus

This make it show up in the right click menu, moving it to admin or adminbus would make it pretty inconvenient.